### PR TITLE
Add MongoDB Alerts Remedy Doc

### PR DESCRIPTION
**Pull request by faem**
Signed-off-by: Mohammad Fahim Abrar <fahimabrar@appscode.com>

### CODE REVIEW

File: charts/mongodb/remedy.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+## TODOs on MongoDB Critical Alerts
+
+### Database Alerts
+
+- #### MongoDBDown
+  - Describe the `MongoDB` CR, check the reason from conditions and try restarting the pods
+  - Check database pod logs
+  - Contact AppsCode team
+- #### MongodbReplicationLag
```
Comment (ArnobKumarSaha) [inline charts/mongodb/remedy.md:9]: All the `Mongodb` should be `MongoDB`

File: charts/mongodb/remedy.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  - Check if there is any network issue which is causing the replication lag.
+- #### MongodbTooManyConnections
+  - Increase `net.maxIncomingConnections` value using [MongoDBOpsRequest](https://kubedb.com/docs/latest/guides/mongodb/reconfigure/overview/)
+- #### MongodbVirtualMemoryUsage
+  - Increase storage and/or memory using [VerticalScaling OpsRequest](https://kubedb.com/docs/latest/guides/mongodb/scaling/vertical-scaling/overview/) and [VolumeExpansion OpsRequest](https://kubedb.com/docs/latest/guides/mongodb/volume-expansion/overview/)
```
Comment (ArnobKumarSaha) [inline charts/mongodb/remedy.md:14]: We can split this alert into two, like `MongoDBVirtualMemoryUsage` & `MongoDBStorageUsage` as referred opsReq is different.

File: charts/mongodb/remedy.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  - Increase Memory using [VerticalScaling OpsRequest](https://kubedb.com/docs/latest/guides/mongodb/scaling/vertical-scaling/overview/)
+  - Index your data properly
+- #### MongodbNumberCursorsOpen
+  - Check for slow queries using `db.currentOp()` command
+  - Check if application that is using mongodb is closing cursors properly
+- #### MongoDBHighLatency | MongodbCursorsTimeouts | MongoDBRecurrentCursorTimeout | MongoDBHighTicketUtilization
+  - Check for slow queries using `db.currentOp()` command
+  - Check database pod logs
+  - Check for network and disk performance
+  - Contact AppsCode team if this issue persists for a long time
```
Comment (ArnobKumarSaha) [inline charts/mongodb/remedy.md:25]: "for long time" is not definitive.  Using "for more than 1 hour" or something like that, would be better.

File: charts/mongodb/remedy.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  - [Describe the BackupSession](https://stash.run/docs/latest/guides/troubleshooting/how-to-troubleshoot/#describe-the-backupsession)
+  - Check the conditions in the BackupSession
+  - Check the reasons of the `false` conditions (if any)
+  - Check the events of the BackupSession
+  - [View the Backup Job/Sidecar log](https://stash.run/docs/latest/guides/troubleshooting/how-to-troubleshoot/#view-backup-jobsidecar-log)
+  - Check if the `INTEGRITY` of Repository is true
+  - [Check the Stash operator log](https://stash.run/docs/latest/guides/troubleshooting/how-to-troubleshoot/#check-stash-operator-log)
+  - Contact AppsCode team
+- #### RestoreSessionFailed
+  - [Describe the RestoreSession](https://stash.run/docs/latest/guides/troubleshooting/how-to-troubleshoot/#describe-the-restoresession)
```
Comment (ArnobKumarSaha) [inline charts/mongodb/remedy.md:65]: change to `MongoDBRestoreSessionFailed` like the other alerts naming-convention

**Review by ArnobKumarSaha — CHANGES_REQUESTED** [review]

File: charts/mongodb/remedy.md (hunk 1 above)
Comment (faem) [reply charts/mongodb/remedy.md:9]: The alerts are already named liked that

File: charts/mongodb/remedy.md (hunk 2 above)
Comment (faem) [reply charts/mongodb/remedy.md:14]: Virtual memory is the summation of storage and memory. I think, in this case, the user needs to decide whichever they want to increase.

Comment (faem) [reply charts/mongodb/remedy.md:14]: `MongoDBLowStorage` should be another alert. We need to add that, maybe in the future.

File: charts/mongodb/remedy.md (hunk 1 above)
Comment (ArnobKumarSaha) [reply charts/mongodb/remedy.md:9]: some of them with `db`, & some others with `DB` ? That's strange !!

**Review by ArnobKumarSaha — APPROVED** [review]

**Pull request merged**